### ISSUE TITLE
Visibility of 'replaced by' on Foundry page

### ISSUE THREAD

**Issue by nataled** [OBOFoundry/OBOFoundry.github.io#431]
On the OBO Foundry page there are a number of ontologies listed as obsolete. Right now, if there is a suitable other ontology in the same domain, that information is buried one click away under 'details'. I think it would be useful to make this directly visible.

**Comment by nlharris** [comment]
I can see the argument for that; on the other hand, "details" reveals a bunch of other fields, and the replacement ontology is just one of those.

**Comment by nataled** [comment]
True, though one could argue that--for obsolete ontologies--what other ontology would be of most interest is probably the KEY piece of information. I don't feel strongly about it, though I suspect if this was directly displayed we would probably have more cases of replacement ontologies indicated.

**Comment by nlharris** [comment]
Is this still relevant and desired?

**Comment by matentzn** [comment]
I will put it on the list for @ddooley team - this should be easy to resolve. (I am thinking mouseover tooltip type solution).

**Comment by jsstevenson** [comment]
I'm looking at this for the first time, but my snap judgment would be something like

* a slightly more elegant way of indication obsolescence (for example, a badge)
* tooltip as @nlharris says, triggered by mouseover on obselete indicator

5 minute mockup (with existing styling for comparison):

<img width="1180" height="368" alt="Image" src="https://github.com/user-attachments/assets/1ca97e29-1dad-4556-8d90-b33a89943a38" />
